test/controllers/deploys_controller_test.rb

```diff
@@ -509,15 +509,15 @@ def pull_request_stub(data_overrides = {})
 
       describe "with a deploy not owned by the user" do
```

Comment:
I believe this test should change because now a user who is not the owner of the deploy can cancel a deploy - any deployer of the project should be able to cancel any deploy within the project, regardless of who owns it.

Reply:
hmmm yes, but from the perspective of `what if this code never existed` the test feels redundant ... and the test we'd need would be `as a viewer I cannot cancel` ... but as ryan said below, this can all go away :)